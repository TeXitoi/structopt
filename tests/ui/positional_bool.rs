use structopt::StructOpt;

#[derive(StructOpt, Debug)]
struct Opt {
    verbose: bool,
}

fn main() {
    Opt::from_args();
}