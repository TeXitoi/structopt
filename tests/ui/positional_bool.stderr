error: `bool` cannot be used as positional parameter with default parser

  = help: if you want to create a flag add `long` or `short`
  = help: If you really want a boolean parameter add an explicit parser, for example `parse(try_from_str)`
  = note: see also https://github.com/TeXitoi/structopt/tree/master/examples/true_or_false.rs

 --> $DIR/positional_bool.rs:5:14
  |
5 |     verbose: bool,
  |              ^^^^
